examples/v2/publisher.nim

```diff
@@ -33,7 +34,7 @@ proc setupAndPublish() {.async.} =
     setupLogLevel(logging.LogLevel.NOTICE)
     notice "starting publisher", wakuPort=wakuPort, discv5Port=discv5Port
     let
-        nodeKey = crypto.PrivateKey.random(Secp256k1, crypto.newRng()[])[]
+        nodeKey = crypto.PrivateKey.random(Secp256k1, wakuCrypto.getRng()[])[]
```

Comment:
Perhaps we can leverage the occasion to start using `generateKey()` util function and saving some lines of code? Would be great to replace all occurrences.

node = WakuNode.new(generateKey(), ip, Port(wakuPort))